tests/gis_tests/tests.py

```diff
@@ -94,3 +94,19 @@ def test_no_version_number(self):
         ops = FakePostGISOperations()
         with self.assertRaises(ImproperlyConfigured):
             ops.spatial_version
+
+    def test_version_dependent_funcs(self):
+        """
+        Test that PostGIS version dependent functions are switched accordingly.
```

Comment:
The guidelines from "[Coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style)" are: In test docstrings, state the expected behavior that each test demonstrates. Don’t include preambles such as “Tests that” or “Ensures that”.

Could you also add something like "Remove when dropping support for PostGIS 2.1"